src/styles/theme.js

```diff
@@ -2,14 +2,16 @@
 import React from 'react';
 import type { Context } from 'react';
 
+import type { ThemeName } from '../reduxTypes';
+
 export type ThemeColors = {|
   color: string,
   backgroundColor: string,
   cardColor: string,
   dividerColor: string,
 |};
 
-export const themeColors: { [string]: ThemeColors } = {
+export const themeColors: { [name: ThemeName | 'light']: ThemeColors } = {
```

Comment:
If you mean 'light' as it occurs in the one line I changed, here, yes: the object literal assigned to `const themeColors` contains a `light` property.

If you're referring to the `light` property, below my diff, yeah, I don't see any reason this shouldn't just be `default`. Looking at 6d5cf6044^, there used to be an object-as-map called `themeNameToObject`, but, at least at that commit, it was only used in one place, and, there, an index of type ThemeName was used on it, and ThemeName has always and forever been 'default' | 'night'.

Reply:
Hmm. Taking a look at Apple's HIG document on Dark Mode, though, it makes it sound like the "default" semantics shouldn't apply to either theme at the application level, but rather at the device level:

> In Settings, people can choose Dark Mode as their default interface style and schedule automatic changes between the appearance modes. Because people make these choices at a systemwide level, they generally expect all apps to respect their preferences.

Perhaps we might actually deprecate the "default" name, and use "night" or "light" everywhere, at or on the path to https://github.com/zulip/zulip-mobile/issues/4009.

Reply:
> Perhaps we might actually deprecate the "default" name, and use "night" or "light" everywhere (with our `ThemeName` type being `'light' | 'night'` instead of `'default' | 'night'` as it is currently), at or on the path to #4009.

That reasoning makes sense to me.

Reply:
Just mentioned this plan at https://github.com/zulip/zulip-mobile/issues/4009#issuecomment-619280681.